docs/examples/Parameters/Parameters.ipynb

```diff
@@ -95,6 +103,10 @@
    "metadata": {},
    "source": [
     "## ArrayParameter\n",
+    "**NOTE:** This is an older base class for array-valued parameters. For any new driver we strongly recommend using `ParameterWithSetpoints` class which is both more flexible and significantly easier to use. Refer to notebook on [writing drivers with ParameterWithSetpoints](http://qcodes.github.io/Qcodes/examples/Parameters/Simple-Example-of-ParameterWithSetpoints.html). \n",
```

Comment:
Please use direct links to other notebooks as documented here https://nbsphinx.readthedocs.io/en/0.4.3/markdown-cells.html#Links-to-Other-Notebooks